tests/test_conditionals.py

```diff
@@ -128,3 +130,40 @@ def test_gaussian_whiten(Xdata, Xnew, kernel, mu, sqrt):
 
     assert_allclose(mean_diff, 0, atol=4)
     assert_allclose(var_diff, 0, atol=4)
+
+@pytest.mark.parametrize('white', [True, False])
+def test_q_sqrt_constraints(Xdata, Xnew, kernel, mu, white):
```

Comment:
What's about the previous code, would this test pass?

Reply:
no